app-quizz/src/app/components/Button/button.tsx

```diff
@@ -1,14 +1,25 @@
+import classNames from "classnames";
+
 type ButtonProps = {
   title: string;
+  style?: React.CSSProperties;
   handlerEvent: () => void;
   btnStyles?: string;
 };
 
 const Button = (props: ButtonProps) => {
-  const btnStyle = props.btnStyles ? props.btnStyles : "bg-teal-900 hover:bg-teal-800 text-white font-bold rounded";
+  const {style} = props;
+  const btnStyles =
+    "bg-teal-900 hover:bg-teal-800 text-white font-bold py-2 px-4 rounded ";
+
+  const classes = classNames(btnStyles, style);
 
   return (
-    <button className={`${btnStyle} px-5 py-2.5 ml-2 mb-2 hover:bg-teal-700`} onClick={props.handlerEvent}>
+    <button
+      className={btnStyles}
```

Comment:
S: I think you can utilize className={classes} here instead of the style prop, making it unnecessary.